### ISSUE TITLE
`process.env` is not defined for `@stoplight/mosaic`

### ISSUE THREAD

**Issue by baseplate-admin** [stoplightio/elements#2412]
## Context

So i have this code : 

```svelte
<script lang="ts">
    import "@stoplight/elements/styles.min.css";

    import { onMount, } from "svelte";
    import React from "react";
    import { API } from "@stoplight/elements";
    import { createRoot } from "react-dom/client";

    const e = React.createElement;
    class Documentation extends React.Component {
        render() {
            return e(
                "div",
                { className: "w-screen" },
                e(API, {
                    apiDescriptionUrl: "/openapi/schema.yaml"
                })
            );
        }
    }
    let container: HTMLDivElement;
    onMount(() => {
        createRoot(container).render(e(Documentation));
    });
</script>



<main>
    <div bind:this={container} />
</main>

```

This is throwing `process is not defined` error

## Current Behavior

So we have this code in `use-safe-effect`. 
```typescript
export const useSafeEffect = typeof window !== 'undefined' && !process.env.TEST_SSR ? useLayoutEffect : useEffect;
```

This is causing issues when `process` is not defined. i.e : in a `sveltekit` app
![image](https://github.com/stoplightio/elements/assets/61817579/1868801a-ca78-458b-9ec6-29bbe7402340)

## Expected Behavior
I want to mount `react` inside `sveltekit` app and expect it to work normally

## Possible Workaround/Solution

Use web-component ( not a big fan cause i am already using react inside my app and the bundle size just gets larger )

## Steps to Reproduce

Live example : https://core-project-v3-ui-git-fix-react-tokitou-san.vercel.app/openapi/stoplight

## Environment

- Version used: 7.13.0
- Environment name and version (e.g. Chrome 39, node.js 5.4): Firefox 119.0a1
- Operating System and version (desktop or mobile): Windows 11 ( desktop )
- Link to your environment/workspace/project: https://github.com/baseplate-admin/CoreProject-v3-ui/pull/796

**Comment by Lexy2** [comment]
I ended up bundling stoplight-elements component in my application using webpack.
Missing dependencing are handled by [NodePolyfillPlugin](https://www.npmjs.com/package/node-polyfill-webpack-plugin).

**Comment by chohmann** [comment]
A fix for this was released in version 7.13.9.
